Update UI colour

Huongg opened the pull request:
## Description

Resolves #1328 

## Development notes

Light theme option 1 [design link](https://www.figma.com/file/K50WL8AfBsuqHREHNJkbcM/Kedro-Design%3A-(IN)-UI-colour-update?type=design&node-id=7026-150374&t=zM2kqkjJ3mxPSwzK-0). Changes are to use the existing white palette with stroke lines
Dark theme option 1 [design link](https://www.figma.com/file/K50WL8AfBsuqHREHNJkbcM/Kedro-Design%3A-(IN)-UI-colour-update?type=design&node-id=6986-44319&t=R5OqzehJN6qAKUcp-0). Changes are to update all the existing slate palette

## QA notes
Things are missing:
- New kedro logo
- Minimap colour for dark theme
- For light theme in flow chart, do we need the blue line for active/selected node as there isn't one for the dark theme. 
- Release note

<!-- How has the expected behaviour changed? What testing strategies have you used? -->

## Checklist

- [x] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes


<a href="https://gitpod.io/#https://github.com/kedro-org/kedro-viz/pull/1359"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>

Mackay031 commented:
Hi, looking good so far. I have a couple of quick comments. I have taken screen shots and will add comments to each. This is a quick first round of comments of issues I immediately noticed. I will take a more in depth look today.

<img width="236" alt="Screenshot 2023-05-16 at 11 10 03" src="https://github.com/kedro-org/kedro-viz/assets/102358798/51f13bac-515d-4dd9-87c2-f23c7064d80a">

The box in the hover state overlaps the keyline


<img width="361" alt="Screenshot 2023-05-16 at 11 06 46" src="https://github.com/kedro-org/kedro-viz/assets/102358798/b8ff8304-7487-41ff-92c5-34f08700bf2c">

The background of the search section should be the TESTslate-600




<img width="382" alt="Screenshot 2023-05-16 at 11 25 43" src="https://github.com/kedro-org/kedro-viz/assets/102358798/5436b425-75e9-4242-ba67-bdae3efdfc4e">

The background of the search section should be the white-100




<img width="199" alt="Screenshot 2023-05-16 at 11 05 12" src="https://github.com/kedro-org/kedro-viz/assets/102358798/2fdf970c-70c9-4ea2-b45a-6eb23f6ea98f">

Can we make the blue highlight the same fill and stroke as the dark theme


<img width="346" alt="Screenshot 2023-05-16 at 11 32 16" src="https://github.com/kedro-org/kedro-viz/assets/102358798/5a321a44-7077-43c9-9338-f45f2174ff94">

The group highlight state in the list is a darker colour. Can we make it slate-200
(#242526) to match the other highlight state



<img width="375" alt="Screenshot 2023-05-16 at 11 39 07" src="https://github.com/kedro-org/kedro-viz/assets/102358798/1323471b-0560-455c-82fc-13b0cc226dad">

Can we add a white-300 keyline at the bottom of the 'all' divider section and all other divider section. If the top run is selected as in this example, then there is no well defined differentiation. This is what I mentioned yesterday on the call.

<img width="905" alt="Screenshot 2023-05-16 at 11 02 24" src="https://github.com/kedro-org/kedro-viz/assets/102358798/47a04389-f15c-4deb-8a09-b146d074b43a">


Has the code block always scrolled right i.e the code is cut off. Maybe be it as...


Thanks!

Mackay031 commented:
Regarding this point:

The background of the search section should be the white-100. I've checked on my code and in the browser, its white-100: #F9F9FA

<img width="865" alt="Screenshot 2023-05-16 at 16 02 19" src="https://github.com/kedro-org/kedro-viz/assets/102358798/419093c5-1cd1-4b12-9156-bae197148796">

In the above screenshot from the Figma file, the 'search' background is the same shade as the list items. If the 'search' background is different, then maybe the list items and runs list background are not white-100 (#F9F9FA0? Could you check? Thanks :)

Huongg commented:
hey @Mackay031, for the final point, its now fixed. They were all the same colour but there was a shadow box from the scrollbar in between thats why it makes it looks like it looks different. I've removed that shadow now.

rashidakanchwala reviewed (commented):
I have only looked at flowchart in light theme so far. Here are some design deviations I found. 


1. there's some left and bottom padding in the design for minimap which is missing. 


<img width="483" alt="Screenshot 2023-05-19 at 11 41 02" src="https://github.com/kedro-org/kedro-viz/assets/37628668/e9a21fa1-0a82-42a1-9261-4d32af255123">

2. The color of show code [blue] is different from design. 


<img width="507" alt="Screenshot 2023-05-19 at 11 39 57" src="https://github.com/kedro-org/kedro-viz/assets/37628668/8399c761-4e56-46ad-80f4-80fd931c1f58">

3. The Tags and Parameters have a different color (darker shades of grey in design) 



<img width="547" alt="Screenshot 2023-05-19 at 11 38 42" src="https://github.com/kedro-org/kedro-viz/assets/37628668/88963af9-ce53-408f-a578-32b6d5aca6df">

<img width="594" alt="Screenshot 2023-05-19 at 11 37 33" src="https://github.com/kedro-org/kedro-viz/assets/37628668/5dd594b5-a8e8-45b9-88f2-90bd2feae326">

Huongg commented:
thanks @rashidakanchwala @Mackay031 , I believed I've addressed the combination of both of your feedback. There are some left that I need to discuss with @Mackay031 to double check with him. Let me know if you're free today or tomorrow to talk about it 😄

tynandebold reviewed (commented):
The code looks good! No comment from me there.

I do have some comments on the colors, specifically the dark theme. On the screen I'm using at the office (ThinkVision monitor), it's hard for me to distinguish between certain elements on the page, those being:

- The nodes on the flowchart and the background
  - Previously our contrast ratio here was 1.6 and now it's 1.29, a ~20% decrease (doesn't matter that it's failing the test for text, as I'm using this more as a comparison of numbers)
    ![Screenshot 2023-05-23 at 14 38 14](https://github.com/kedro-org/kedro-viz/assets/16869061/9e862889-4300-4ccc-a7ea-9f53d5a21de8)
- The modals (settings and edit exp. tracking run). There's barely any contrast there.
- In experiment tracking, it's hard to see any difference between the run metadata and run tracking data section as well as the tabs: 
    ![image](https://github.com/kedro-org/kedro-viz/assets/16869061/e0f3be7f-d538-4895-bda3-24c06473118f)

I want to make sure we're checking these colors on non-retina and non-4k screens. Are we? When I look at the update on my MacBook Pro I can see things much clearer, but we can't assume all or even most of our users are using that hardware.

Lastly, the light theme looks amazing, even on this monitor at the office. I've always liked it, and now like it even more.

cc @Mackay031 @stephkaiser

rashidakanchwala reviewed (approved):
Just reviewed the code, not the UI and it all looked fine. thanks Huong

Huongg commented on src/components/experiment-tracking/parallel-coordinates/parallel-coordinates.js:
```diff
@@ -24,7 +24,7 @@ const paddingLeftRight = 80;
 const selectedMarkerRotate = [45, 0, 0];
 
-const selectedMarkerColors = ['#0084B2', '#FFBC00', '#31E27B'];
+const selectedMarkerColors = ['#A000BC', '#FFBC00', '#31E27B'];
```
hey yeah good shout, I've moved them to css now so we can re-use the variable

tynandebold reviewed (approved):
The code LGTM as well.

Huongg commented:
hey @tynandebold to confirm, @Mackay031 and I have addressed point 1 and 2 from your comments. Point 3 will be done in the future as it requires more thoughts and changes on that
